backend/extensions/Squidex.Extensions/Actions/SignalR/SignalRAction.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+using Squidex.Domain.Apps.Core.HandleRules;
+using Squidex.Domain.Apps.Core.Rules;
+using Squidex.Infrastructure.Validation;
+
+namespace Squidex.Extensions.Actions.SignalR
+{
+    [RuleAction(
+        Title = "Azure SignalR",
+        IconImage = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 32 32'><path d='M.011 16L0 6.248l12-1.63V16zM14 4.328L29.996 2v14H14zM30 18l-.004 14L14 29.75V18zM12 29.495L.01 27.851.009 18H12z'/></svg>",
+        IconColor = "#1566BF",
+        Display = "Send to Azure SignalR",
+        Description = "Send an message to azure SignalR.",
+        ReadMore = "https://azure.microsoft.com/fr-fr/services/signalr-service/")]
+    public sealed record SignalRAction : RuleAction
+    {
+        [LocalizedRequired]
+        [Display(Name = "Connection", Description = "The connection string to the Signal R Azure.")]
+        [DataType(DataType.Text)]
+        [Formattable]
+        public string ConnectionString { get; set; }
+
+        [LocalizedRequired]
+        [Display(Name = "Hub Name", Description = "The name of the hub.")]
+        [DataType(DataType.Text)]
+        [Formattable]
+        public string HubName { get; set; }
+
+        [LocalizedRequired]
+        [Display(Name = "Action", Description = "BROADCAST -> send to all User, USER => send to one user, USERS => send to list of users, GROUP => send to group, GROUPS => send to list of groups.")]
```

Comment:
You are using different arrows here. I am not sure, but I think you can also use markdown here.

Reply:
I change the first arrows, markdown doesn't seem to work
![image](https://user-images.githubusercontent.com/4747563/127933882-33eb5cfd-3fb9-48df-b74b-ec0b856525a8.png)